util/src/main/java/io/kubernetes/client/informer/cache/Deltas.java

```diff
@@ -0,0 +1,77 @@
+package io.kubernetes.client.informer.cache;
+
+import io.kubernetes.client.util.common.Collections;
+import java.util.List;
+
+/**
+ * Deltas class is the queue'd entry in the DeltaFIFO class. It consists of one or more Delta class
+ * which records the operation type and its state of the object.
+ */
+public class Deltas<ApiType> {
```

Comment:
This is basically exactly a [Java `Deque`](https://docs.oracle.com/javase/7/docs/api/java/util/Deque.html)

Let's remove the `Deltas` class and just use `Deque`? (or at the very least use a `Deque` instead of a `List`)

Reply:
> This is basically exactly a Java Deque

yesssssss. that's a great point. replaced w/ Deque